tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -21,7 +21,7 @@
 from sentry.testutils.skips import requires_snuba
 from sentry.utils.samples import load_data
 
-pytestmark = [requires_snuba]
+pytestmark = [pytest.mark.sentry_metrics, requires_snuba]
```

Comment:
What does this do?

Reply:
any attempt to access metrics in tests seems to fail without it